services/apps/alcs/src/common/automapper/application-subtask.automapper.profile.ts

```diff
@@ -47,9 +45,7 @@ export class ApplicationSubtaskProfile extends AutomapperProfile {
         HomepageSubtaskDTO,
         forMember(
           (a) => a.completedAt,
-          mapFrom((ad) =>
-            ad.completedAt ? ad.completedAt.getTime() : undefined,
-          ),
+          mapFrom((ad) => ad.completedAt?.getTime()),
```

Comment:
Not sure, but I think the previous version would prevent deleting data from db if completedAt is sent as null, where the current will allow that.